src/hooks/useCheckRevision.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import useFocusInput from './useFocusInput';
+
+const useCheckRevision = () => {
+  const { handleChildClick } = useFocusInput();
+  const [checked, setChecked] = React.useState<string[]>([]);
```

Comment:
Probably this is not a huge performance win (since this state array will be very small), but we could also use `Set` object here, assuming order of this array is not important.